stampit.js

```diff
@@ -45,7 +45,7 @@ function extractFunctions(arg) {
 
 function addMethods(fixed, methods) {
   var args = [fixed.methods].concat(methods);
-  mixer.mixinFunctions.apply(null, args);
+  mixer.mixin.apply(null, args);
```

Comment:
The `hacking.js` example prints `undefined`. Whereas it should print 
```
No credentials were given:  { name: 'guest', password: 'guest' }
Credentials were given:  { name: 'admin', password: '123' }
Reusing the default credentials:  { name: 'guest', password: 'guest' }
```

Please. fill free to push your ideas to this branch.